packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -303,6 +318,23 @@ class Auth {
         } else {
             dntCookieStatus = Boolean(Number(dntCookieVal))
         }
+
+        if (options?.useDefaults) {
```

Comment:
Let's avoid the word `use...` since it may get confused with react hook naming convention. 
```suggestion
        if (options?.includeDefaultDnt) {
```

Reply:
I'll make this `includeDefaults` because it includes not only defaultDnt but also the PWA Kit Default if defaultDnt is not defined

Reply:
what pwa-kit default are you mentioning? I thought that pwa-kit default is becoming defaultDnt value when it is a available, no? From commerce-sdk-react, there should be only defaultDnt 🤔

Reply:
defaultDnt might not be defined, so there should be another default that takes its place. This PWA kit default mirrors the [SLAS default, where if dnt claim is not defined in the access token, dnt is by default false in ECOM](https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-login?meta=getAccessToken).

However, in the code, I just have all calls to the access token endpoint include the dnt claim for clarity.